### DISCUSSION TITLE
Is it just me or is the preheat menu far down the menu list? Might be better at the top somewhere for convenience.

### DISCUSSION THREAD

**Discussion by Bram-Brooks** [Jyers/Marlin#722] [Ideas]
I find myself often using the preheat menu but have to scroll noticeably much when it would seem like a much used feature. It might make more sense to move it up higher in the menu list for easy access.

**Comment by Cal-LF** [comment]
I think the position reflects where it is in the main Marlin FW. I may be wrong.

**Comment by tiagofreire-pt** [comment]
Yes, I suffer with that pain a lot. Up voting this one. 👍
